edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/Microsoft.Azure.Devices.Edge.Hub.Amqp.csproj

```diff
@@ -8,8 +8,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.7" />
-    <PackageReference Include="Microsoft.Azure.Devices" Version="1.31.1" />
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.5.10" />
```

Comment:
Callout:
We need to bump this because 1.31.2 bumped Microsoft.Azure.Amqp to 2.5.10. This then resulted in a necessary code change to EdgeAmqpConnection.cs to change one of the timeout variables.

When trying to use `this.underlyingAmqpConnection.DefaultCloseTimeout`, it gives a deprecation error and says to use `this.underlyingAmqpConnection.Settings.OperationTimeout`. I assume these are the same, but they aren't consts so it wasn't obvious. The E2E tests at least pass with this change.

@vipeller Do you think this is fine?